core/README.md

```diff
@@ -80,11 +94,18 @@ timestamp of the actual execution.
 > We define more temporal properties in the
 > [timestamps extension](https://github.com/fiboa/timestamps).
 
-**determination_method**: Allowed values:
-- `manual` (hand drawn from imagery)
-- `driven`
-- `surveyed`
-- `administrative`
-- `auto-operation` (auto-created from operation)
-- `auto-imagery` (auto-created from imagery)
-- `unknown`
+**determination_method**: Must be one of the following values:
```

Comment:
It'd also be good here to refer to somewhere more 'standard' in adapt / ag gateway than just listing here. But we may need to wait for adapt to do something here. Hopefully we can link to at least a github issue?